docs/writing-stories/naming-components-and-hierarchy.md

```diff
@@ -103,7 +103,7 @@ The `storySort` can also accept a configuration object.
 | Field           |  Type   |                       Description                        | Required |      Default Value      |          Example          |
 | --------------- | :-----: | :------------------------------------------------------: | :------: | :---------------------: | :-----------------------: |
 | **method**      | String  | Tells Storybook in which order the stories are displayed |    No    | Storybook configuration |     `'alphabetical'`      |
-| **order**       |  Array  |     The stories to be show, ordered by supplied name     |    No    |    Empty Array `[]`     | `['Intro', 'Components']` |
+| **order**       |  Array  |     The stories to be shown, ordered by supplied name     |    No    |    Empty Array `[]`     | `['Intro', 'Components']` |
```

Comment:
We could even update to "The stories to show" to use active voice. Thanks for updating this!

Reply:
Indeed @darleendenno sounds and reads better that way, appreciate the feedback and improvements.